### ISSUE TITLE
Snipe-IT changes User during work

### ISSUE THREAD

**Issue by OH-RV-MG-2024** [grokability/snipe-it#15399]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

After logging into the Snipe-IT console, it often happens that I am no longer logged in as myself, but as another user who is currently working in Snipe-IT. Not only does the user change, but also the rights associated with the new user. We have had this error for some time and had hoped it would disappear after an update. Unfortunately, this does not happen. We have already searched and tried everything, unfortunately without success. Does anyone here have any ideas as to what could be causing this, or has experience with this error? It would be great to receive feedback.

### Reproduction steps

1. Log in with your account
2. Work with snipe-it
3. See that the user chganges
...

### Expected behavior

I expect that during my work with Snipe-IT the logged in user remains the same until I log out.

### Snipe-IT Version

v7.0.11 build 15044 (g46ed07642) 

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

8.1.2-1ubuntu2.18 

### Operating System

Ubuntu

### Browser

Firefox

### Version

129.0.2

**Comment by snipe** [comment]
Is it possible the browser is storing the credentials and auto-logging you in as the other user?

**Comment by snipe** [comment]
What do you have in the `# OPTIONAL: SESSION SETTINGS` section of your `.env`?

**Comment by OH-RV-MG-2024** [comment]
No, that can't be the problem. We have also deleted the browser cache several times and used different browsers, unfortunately without success.

**Comment by OH-RV-MG-2024** [comment]
We log in with the user name and password. Then we use the 2 FA with the Google Authenticator.
Or what is meant by this

**Comment by OH-RV-MG-2024** [comment]
> Is it possible the browser is storing the credentials and auto-logging you in as the other user?

No, that can't be the problem. We have also deleted the browser cache several times and used different browsers, unfortunately without success.
We all work via RDP and the user you are suddenly logged in with is working at a completely different location, for example. What is noticeable, however, is that when you are alone in Snipe, everything is fine. Just as the other user logs in at a different location, it then happens that the user name switches to your own.

**Comment by OH-RV-MG-2024** [comment]
> What do you have in the `# OPTIONAL: SESSION SETTINGS` section of your `.env`?

We log in with the user name and password. Then we use the 2 FA with the Google Authenticator.
Or what is meant by this

**Comment by snipe** [comment]
What do you have in the # OPTIONAL: SESSION SETTINGS section of your .env?

**Comment by snipe** [comment]
Are you using php-fpm in a chrooted environment by any chance?

**Comment by Contused** [comment]
I recently set up snipe via docker compose and have a similar issue:
When I log in with the admin account and my colleague signs in with his account afterwards, he will also be logged in the admin account. I tried changig the settings in the .env to:

# --------------------------------------------
# OPTIONAL: SESSION SETTINGS
# --------------------------------------------
SESSION_LIFETIME=12000
EXPIRE_ON_CLOSE=false
ENCRYPT=true
COOKIE_NAME=snipeit_session
COOKIE_DOMAIN=null
SECURE_COOKIES=true
API_TOKEN_EXPIRATION_YEARS=40

But that did not resolve the issue.
Details:
-Ubuntu 24.04.2 LTS
-HTTPS hosted with Nginx reverse proxy
-installed following the docker (compose) tutorial

Is there a solution to this problem/ a setting I overlooked?

**Comment by snipe** [comment]
Are you running this via proxy or reverse proxy or anything else unusual? We’ve seen this before (very rarely) and it’s usually a network config

**Comment by Contused** [comment]
Yes, we are using an Nginx Reverse Proxy. Is there a configuration on which we can run snipe it does it not work at all with a reverse proxy?

**Comment by snipe** [comment]
It definitely does work with a reverse proxy. Try taking a look at this issue - https://github.com/grokability/snipe-it/issues/15495

**Comment by Contused** [comment]
Thank you. I'll take a look!

**Comment by Contused** [comment]
Disabling "Cache Assets" in the Nginx Proxy Manager WebUI (as mentioned in #15495) seemed to work. I'll keep an eye on it. Thank you very much for your support!

**Comment by snipe** [comment]
Thanks so much for the additional info!
